pipelines/live-1/main/check-environment.yaml

```diff
@@ -44,114 +34,88 @@ jobs:
     - aggregate:
       - get: every-week
         trigger: true
-      - get: cloud-platform-environments-repo
-      - get: tools-image
+      - get: ruby-image
     - task: check-environments
-      image: tools-image
+      image: ruby-image
       config:
         platform: linux
         params:
-          AWS_ACCESS_KEY_ID: ((aws-live-0.access-key-id))
-          AWS_SECRET_ACCESS_KEY: ((aws-live-0.secret-access-key))
+          BUCKET_PREFIX: cloud-platform-live-0.k8s.integration.dsd.io
           KUBECONFIG_AWS_ACCESS_KEY_ID: ((aws-live-1.access-key-id))
           KUBECONFIG_AWS_SECRET_ACCESS_KEY: ((aws-live-1.secret-access-key))
+          KUBECONFIG_AWS_REGION: eu-west-2
+          KUBECONFIG_S3_BUCKET: cloud-platform-concourse-kubeconfig
+          KUBECONFIG_S3_KEY: kubeconfig
+          KUBECONTEXT: cloud-platform-live-0.k8s.integration.dsd.io
           KUBECONFIG: /tmp/kubeconfig
-          TF_PLUGIN_CACHE_DIR: /tmp/terraform-plugin-cache
-          # the variables prefixed with PIPELINE_ are used by the apply script
           PIPELINE_CLUSTER: cloud-platform-live-0.k8s.integration.dsd.io
```

Comment:
From a quick scan of the code, `BUCKET_PREFIX`, `PIPELINE_STATE_BUCKET` and `TFSTATE_*` are used to reach the state files of environments.

However, `PIPELINE_CLUSTER` is used to find the relevant environment files on github.

If my understand is correct, I think we should choose a common prefix so that the intent is clear and it's easier to verify/update the variables in the future?

Reply:
Changed the env vars as 644a595